views/forms/error-summary/code.html.twig

```diff
@@ -0,0 +1,135 @@
+
+    {{ docs.example({
+        screenreader: "",
+        twig: "
+    {{
+        form.create()
+    }}
+
+    {{
+        form.error_summary({
+            'heading': 'There is a problem',
+            'errors': [
+                {
+                    'label': 'Enter your first name',
+                    'href': 'first-name'
+                },
+                {
+                    'label': 'Enter your last name',
+                    'href': 'last-name'
+                }
+            ]
+        })
+    }}
+
+    {{ form.fieldset_start({'legend': 'Example form with errors'}) }}
+
+        {{
+            form.text({
+                'label': 'First name',
+                'id': 'first-name',
+                'required': true,
+                'error': 'Enter your first name'
+            })
+        }}
+
+        {{
+            form.text({
+                'label': 'Last name',
+                'id': 'last-name',
+                'required': true,
+                'error': 'Enter your last name'
+            })
+        }}
+
+    {{ form.fieldset_end() }}
+
+    {{
+        form.end({
+            'actions': [
+                html.button({
+                    'label': 'Save',
+                    'class': 'btn--primary'
+                }),
+                html.button({
+                    'label': 'Cancel',
+                    'class': 'btn--naked'
+                })
+            ]
+        })
+    }}
+    " }) }}
+
+
+    {{ docs.heading('h2', 'Dependencies') }}
+
+    <p>Example code based on common Pulsar/import methods, may need to be modified to suit your needs</p>
+
+    {{ docs.heading('h3', 'index.js') }}
+<pre><code lang="javascript">var ErrorSummaryComponent = require('/path/to/Repeater/ErrorSummaryComponent');
+
+module.exports = {
+    ErrorSummaryComponent
+};</code></pre>
+
+{{ docs.heading('h3', 'main.js') }}
+
+<pre><code lang="javascript">(function ($) {
+    pulsar.errorSummary = new pulsar.ErrorSummaryComponent();
+
+    $(function () {
+        pulsar.errorSummary.init($html);
+    });
+}(jQuery));</code></pre>
+
+{{ docs.heading('h3', 'pulsar.scss') }}
+
+<pre><code lang="scss">// Components
+@import 'component.error-summary';
+</code></pre>
+
+    {{ docs.heading('h2', 'Helper options') }}
+
+    <p>You can configure this helper using the <a href="../guides/using-helpers#common-options">common helper options</a>, the following options are specific to this helper.</p>
+
+    <table class="table table--full">
+        <thead>
+            <tr>
+            <th>Option</th>
+            <th>Type</th>
+            <th>Description</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td>heading</td>
+                <td>string</td>
+                <td>The text to be used as the error summary heading. This should always be "There is a problem".</td>
```

Comment:
Why did we make this configurable if we're telling them it should be a specific value?

Reply:
Translations